shared/templates/template_BASH_regex_permissions

```diff
@@ -0,0 +1,6 @@
+# platform = multi_platform_all, Red Hat Enterprise Linux 5
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+find FILEPATH -regex '^FILEPATH/FILENAME' -exec chmod FILEMODE {} \;
```

Comment:
because the filepath is regex based, not globbing based. chmod is globbing based. If I just update the chmod, the regex will not work out of the box